src/Mod/Spreadsheet/Gui/qtcolorpicker.cpp

```diff
@@ -902,7 +902,12 @@ void ColorPickerPopup::getColorFromDialog()
 {
     //bool ok;
     //QRgb rgb = QColorDialog::getRgba(lastSel.rgba(), &ok, parentWidget());
-    QColor col = QColorDialog::getColor(lastSel,parentWidget(),0,QColorDialog::ShowAlphaChannel);
+    QColor col;
+    if (Gui::DialogOptions::dontUseNativeColorDialog()){
+        col = QColorDialog::getColor(lastSel,parentWidget(),0,QColorDialog::ShowAlphaChannel|QColorDialog::DontUseNativeDialog);
```

Comment:
Minor comment: we use spaces behind commas

I will merge and fix the style later today.

Reply:
seems as admin I can change this directly in the PR